### PR TITLE
docs: zkapp and o1js FAQ starter set

### PR DESCRIPTION
**Pull request by barriebyron**
Starter set of FAQs
zkApp FAQ https://docs2-git-faq-two-minadocs.vercel.app/zkapps/faq

seeking product input @garwalsh  for scope, direction, strategy

part of https://app.zenhub.com/workspaces/zkapps-product-eng-6130fedb3b0fc600123d8796/issues/gh/o1-labs/docs2/460 #460

### CODE REVIEW

File: docs/zkapps/faq.mdx [hunk 1]
```diff
@@ -1,36 +1,56 @@
...
+
+### Where can I ask questions and contribute answers?
+
+[Mina Protocol Discord](https://discord.gg/minaprotocol) is the most popular place where Mina enthusiasts and technical contributors gather. 
+
+Join us in these zkApps channels:
+
+* [#zkapps-developers](https://discord.com/channels/484437221055922177/915745847692636181) to meet other developers building zkApps with o1js
+* [#zkapps-general](https://discord.com/channels/484437221055922177/910549624413102100) to ask general questions about zkApps, how to use a zkApp, and so on
+* [#zkapps-questions](https://discord.com/channels/484437221055922177/1047214314349658172) to ask zkApps-related questions and see Q&A history
```
Comment (garwalsh) [inline docs/zkapps/faq.mdx:30]: @barriebyron These channels are also relevant for o1js as it's the defacto place for developers to ask questions (not just about zkapps)

File: docs/zkapps/o1js/o1js-faq.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+sidebar_label: FAQ
+description: Answers to common questions about o1js, a library that creates zk circuits from user code. Learn about the use of built-in o1js datatypes, functions, and the implications for proof generation.
+keywords:
+  - o1js
+  - FAQ
+---
+
+# o1js FAQ
+
+Answers to common questions about o1js, a TypeScript library that creates zk circuits from user code.
```
Comment (garwalsh) [inline docs/zkapps/o1js/o1js-faq.mdx:12]: a TypeScript library for writing zk smart contracts.

File: docs/zkapps/faq.mdx (hunk 1 above)
Comment (barriebyron) [reply docs/zkapps/faq.mdx:30]: per our convo we will combine into a single FAQ for zkApps and o1js

File: docs/zkapps/faq.mdx [hunk 3]
```diff
@@ -1,36 +1,60 @@
...
 - `<yourcontract>.ts`: Your specific smart contract logic. 
 
-## What is ZkProgram?
+### Where can I find the o1js API reference documentation?
+
+See the autogenerated [o1js reference](/zkapps/o1js-reference) documentation with doc comments, like the [Provable](/zkapps/o1js-reference/interfaces/Provable) module.
+
+### What is ZkProgram?
 
 The API for writing non-Mina related provable code. A ZkProgram is similar to a zkApp smart contract, but isn't tied to an on-chain account.
```
Comment (garwalsh) [inline docs/zkapps/faq.mdx:48]: A general purpose API for creating zk proofs.

Comment (barriebyron) [inline docs/zkapps/faq.mdx:48]:
```suggestion
A general-purpose API for creating zk proofs. A ZkProgram is similar to a zkApp smart contract but isn't tied to an on-chain account. 
```

**Review by garwalsh — APPROVED** [review]

**Pull request merged**